Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Templates/TemplateReplacementSet.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Build.Unity.ProjectGeneration.Templates
 {
     /// <summary>
-    /// This class is used to hold the values for token and template replacements that will be used when the template is written out.
+    /// This class is used to hold the values for toke and template replacements that will be used when the template is writen out.
```

Comment:
Looks like a comment regression from the last PR here (re-adding some misspelled words)

```suggestion
    /// This class is used to hold the values for token and template replacements that will be used when the template is written out.
```

Reply:
My bad :(